packages/material-tailwind-react/package.json

```diff
@@ -1,37 +1,37 @@
 {
-  "name": "@material-tailwind/react",
+  "name": "@allen-career-institute/material-tailwind-react",
   "homepage": "https://material-tailwind.com",
-  "version": "2.1.10",
+  "version": "1.0.0",
```

Comment:
would suggest to keep version number as 2.2.x, put a note in readme that we forked from 2.1.10 version of original library.

Reply:
Yes updated as per recommendation